website/docs/r/pi_shared_processor_pool.html.markdown

```diff
@@ -44,48 +43,50 @@ resource "ibm_pi_shared_processor_pool" "testacc_shared_processor_pool" {
+- `pi_shared_processor_pool_placement_groups` - (Optional, List) The list of shared processor pool placement groups that the shared processor pool is in.
+- `pi_shared_processor_pool_reserved_cores` - (Required, Integer) The amount of reserved cores for the shared processor pool.
+- `pi_user_tags` - (Optional, List) The user tags attached to this resource.
 
 ## Attribute reference
 
- In addition to all argument reference list, you can access the following attribute reference after your resource is created.
+In addition to all argument reference list, you can access the following attribute reference after your resource is created.
 
-* `allocated_cores` - (Float) The allocated cores in the shared processor pool.
-* `available_cores` - (Integer) The available cores in the shared processor pool.
-* `crn` - (String) The CRN of this resource.
-* `host_id` - (Integer) The host ID where the shared processor pool resides.
-* `instances` - (List of Map) The list of server instances that are deployed in the shared processor pool.
+- `allocated_cores` - (Float) The allocated cores in the shared processor pool.
+- `available_cores` - (Integer) The available cores in the shared processor pool.
+- `crn` - (String) The CRN of this resource.
+- `host_id` - (Integer) The host ID where the shared processor pool resides.
+- `instances` - (List of Map) The list of server instances that are deployed in the shared processor pool.
   
   Nested scheme for `instances`:
-  * `availability_zone` - (String) Availability zone for the server instances.
-  * `cpus` - (Integer) The amount of cpus for the server instance.
-  * `id` - (String) The server instance ID.
-  * `memory` - (Integer) The amount of memory for the server instance.
-  * `name` - (String) The server instance name.
-  * `status` - (String) Status of the instance.
-  * `uncapped` - (Bool) Identifies if uncapped or not.
-  * `vcpus` - (Float) The amout of vcpus for the server instance.
-* `shared_processor_pool_id` - (String) The shared processor pool's unique ID.
-* `status` - (String) The status of the shared processor pool.
-* `status_detail` - (String) The status details of the shared processor pool.
+  - `availability_zone` - (String) Availability zone for the server instances.
+  - `cpus` - (Integer) The amount of cpus for the server instance.
+  - `id` - (String) The server instance ID.
+  - `memory` - (Integer) The amount of memory for the server instance.
+  - `name` - (String) The server instance name.
+  - `status` - (String) Status of the instance.
+  - `uncapped` - (Bool) Identifies if uncapped or not.
+  - `vcpus` - (Float) The amout of vcpus for the server instance.
+- `shared_processor_pool_id` - (String) The shared processor pool's unique ID.
+- `spp_placement_groups` - (Deprecated, List) The list of shared processor pool placement groups that the shared processor pool is in.
```

Comment:
I believe we can skip this one since it was not mention before. The user will see the deprecation message when run their configs.

Reply:
ok I won't add it.